docs/Android/ndk-crash.md

```diff
@@ -0,0 +1,117 @@
+---
+layout: page
+title: NDK Crash
+hide:
+  #- navigation
+  - toc
+---
+
+## NDK Crash Reporting
+
+NDK Crash events will be reported to the Intelligence Console on the subsequent app load, after configuration and authentication if needed, following the crash.
+
+These NDK Crash events can be found in the Crash section of the Intelligence Console, and can isolated by table filters such as setting `is_ndk` to true.
+
+## NDK Crash Information
+General Intelligence Crash Event attributes and definitions can be found here: [Crash - Android](https://docs.omnissa.com/bundle/WS1Intelligence/page/IntelExpMngtDefMobileIntelligenceSDK.html#crashes_-_android)
+Key information behind the Native Crash caught will be sent as the `reason` attribute for the Crash Event.
+
+This reason will be structured as a JSON block, containing dump information around the crash.
+
+An example of the NDK crash info appears as such:
+### “MyException” Custom C++ Exception Example:
+```JSON
+// JSON NDK Crash Information
```

Comment:
Looks like the JSON's in this file are erroring out (marked red within the JSON blobs) due to the comment. Could the comment be added outside the JSON?

Reply:
Too bad there are no comments in JSON.